coralproject/talk
# [CORL-3001]: clean up dependencies in client and server post-monorepo update

## What does this PR do?

These changes audit the dependencies for both `client` and `server` post-monorepo updates. We had a lot of dependencies in both when they only needed to be in either `client` or `server`.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Run `sh scripts/clear-node_modules.sh` and `sh npm-i.sh`. Then run Coral and click around, seeing that there are no errors and all still works as expected.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
